frontend/public/locales/en/terms-and-conditions.json

```diff
@@ -0,0 +1,63 @@
+      "terms-and-conditions-of-use-part-four": "Your access to and use of the Online Application, including any and all content, is subject to (1)your acceptance of, and (2) your compliance with these Terms of Use.",
+      "terms-and-conditions-of-use-part-five": "Every time you use the Online Application, these Terms continue to apply. By using the Online Application, you agree to abide by the following Terms. It is your responsibility to understand and retain these Terms.",
+      "terms-and-conditions-of-use-part-six": "You may always decide not to accept the Terms, in which case you will not be permitted to access and use the Online Application.",
+      "terms-and-conditions-of-use-part-seven": "In these Terms, “ESDC” refers to the Minister of Employment and Social Development, representing His Majesty the King in Right of Canada.",
+      "terms-and-conditions-of-use-terms-of-use-of-the-online-application-title": "Terms of Use of the Online Application.",
+      "terms-and-conditions-of-use-terms-of-use-of-the-online-application-bullet-one": "Using the Online Application means that you have read and agree to these Terms of Use and the Privacy Notice Statement. If, at any time, you withdraw your agreement to any of the Terms and Conditions of Use you must stop use of the Online Application; or",
+      "terms-and-conditions-of-use-terms-of-use-of-the-online-application-bullet-two": "If you are using the Application on behalf of another individual, by using the Application, you certify that you have all the necessary authority, permission, and authorities to agree to the Terms of Use on behalf of that individual, and that you agree to the Terms on your own personal behalf and on behalf of the individual you are representing.",
+      "terms-and-conditions-of-use-terms-of-use-of-the-online-application-bullet-three": "You are using this service at your own risk, and you need to keep any personal information safe. Make sure to close the browser or log out when you are done.",
+      "terms-and-conditions-of-use-terms-of-use-of-the-online-application-bullet-four": "Only use the Online Application to apply to the Canadian Dental Care Plan.",
+      "terms-and-conditions-of-use-terms-of-use-of-the-online-application-bullet-five": "There will be times when the system might be down for maintenance.",
+      "terms-and-conditions-of-use-terms-of-use-of-the-online-application-bullet-six": "If you are inactive for 15 minutes on the Online Application, your session will end and you will lose any information you entered.",
+      "terms-and-conditions-of-use-terms-of-use-of-the-online-application-bullet-seven": "ESDC uses services from Microsoft Corporation to run the Online Application. The Microsoft data centers used by ESDC are in Canada in accordance with Microsoft Corporation’s Terms of Service Agreement.",
```

Comment:
Consider renaming these i18n keys to leave out the order number from its name. 

It's possible that the order changes and developers will just change the order in the page/component itself but not rename the i18n key. We could have `bullet-seven` occur before `bullet-five` in this scenario.

This comment can be applied to the i18n keys that include `part-one`, `part-two`, etc. as well.

Reply:
I did it for some of the bullet points (ed62887534ecb1a1d9810f348c88031cfc813d39) but sometimes the locales get funky if I do it for too many.

Could I make a separate PR for the renaming/refactored code?

Reply:
I'm good with that. Resolving.